src/main/java/com/puppycrawl/tools/checkstyle/checks/OrderedPropertiesCheck.java

```diff
@@ -206,7 +206,7 @@ private static int getIndex(int startLineNo, FileText fileText, String keyName)
      */
     private static Pattern getKeyPattern(String keyName) {
         final String keyPatternString = "^" + SPACE_PATTERN.matcher(keyName)
-                .replaceAll(Matcher.quoteReplacement("\\\\ ")) + "[\\s:=].*";
+                .replaceAll("") + "[\\s:=].*";
```

Comment:
@Kevin222004 You can have properties with space in them, eg:
```properties
foo\ bar:hoo
```
The whitespace is escaped. You can see https://stackoverflow.com/q/2108103/15412365

Reply:
Thanks @Vyom-Yadav for your help.